### ISSUE TITLE
[1Password] Error: 'The "file" argument must be of type string. Received undefined'

### ISSUE THREAD

**Issue by joshuahhh** [raycast/extensions#12106]
### Extension

https://www.raycast.com/khasbilegt/1password

### Description

When I use the "My Passwords" action, the search window loads but no searches yield results. An error appears in the bottom-left: 'The "file" argument must be of type string. Received undefined'. (See screenshot.)

<img width="476" alt="Screenshot 2024-04-30 at 4 02 07 PM" src="https://github.com/raycast/extensions/assets/643799/d9b68de0-dc88-48ed-abf9-f39a6580622c">

### Steps To Reproduce

See above. I don't imagine everyone else is having this same problem, but I'm not sure what to dig into to diagnose it. Thanks!

**Comment by Luk4h** [comment]
Hello, please ensure that you have the 1Password CLI installed, and its path is correctly set on the extension settings (check using 'where op' on terminal).

**Comment by joshuahhh** [comment]
Ah! I got that stuff set up, and then I had an "Unexpected end of JSON" error (with no passwords) until I connected 1Password CLI to the desktop app. It's working now, except I still get a (seemingly innocuous) "Unexpected end of JSON" error, as mentioned at https://github.com/raycast/extensions/issues/11826.

I might be mistaken, but I don't think I saw any message about installing the 1Password CLI during my path to using this extension – not on the store page, or in the extension itself, or in the error message. I bet it would be helpful to message this more, perhaps most usefully at the site of the error. Sorry if you're working on this and I'm telling you something you already know! Just my two cents.

Thanks a bunch for your help.
